src/Mail/MailService.php

```diff
@@ -97,6 +98,9 @@ public function message($to, string $title, string $body, array $attachments = [
         $this->em->persist($msgEntity);
 
         foreach ($to as $person) {
+            if (!$person instanceof LocalAccount) {
```

Comment:
Is there a reason we won't want to mail the externals? privacy?
(besides the fact that currently the email is optional for an external registrant)

Reply:
didn't consider that. good point